cni/nnsgrpcclient/nnsgrpclient.go

```diff
@@ -0,0 +1,89 @@
+// Node network service grpc client
+package nnsgrpcclient
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-container-networking/log"
+	nns "github.com/Azure/azure-container-networking/proto/nodenetworkservice/3.302.0.744"
+	"google.golang.org/grpc"
+	"time"
+)
+
+const (
+	nnsPort = "6668"
+	apiTimeout = 5 * time.Minute // recommended timeout from node service
+	connectionTimeout = 2 * time.Minute
```

Comment:
is 2 mins isn't it longer timeout?

Reply:
default http connection timeout is 2 mins, so I kept it same.